verifiers/utils/builder.go

```diff
@@ -138,6 +138,7 @@ func IsValidBuilderTag(ref string, testing bool) error {
 			semver.Build(pin) != "" {
 			return fmt.Errorf("%w: %s: version tag not valid", serrors.ErrorInvalidRef, pin)
 		}
+		return nil
```

Comment:
NICE CATCH! Took me a second myself to figure out why it was going to the non-testing block.